raiden/network/rpc/client.py

```diff
@@ -186,6 +186,7 @@ def __init__(
             gasprice: int = None,
             nonce_update_interval: float = 5.0,
             nonce_offset: int = 0,
+            web3: Web3 = None
```

Comment:
you added a lot of commas, but miss one here

Reply:
I've now added another _large_ bag of commas. All hail the glorious `flake8-commas`.

Reply:
Hehe, unfortunately there's no auto-fix with that :(